src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -556,9 +600,10 @@ private[fastq] case class SampleInfo(sample: Sample, isUnmatched: Boolean = fals
 private[fastq] object FastqDemultiplexer {
 
   /** Stores the minimal information for a single template read. */
-  case class DemuxRecord(name: String, bases: String, quals: String, molecularBarcode: Option[String],
-                         sampleBarcode: Option[String], readNumber: Int, pairedEnd: Boolean, comment: Option[String],
-                         originalBases: Option[String] = None, originalQuals: Option[String] = None)
+  case class DemuxRecord(name: String, bases: String, quals: String, molecularBarcode: Seq[String],
+                         sampleBarcode: Seq[String], readNumber: Int, pairedEnd: Boolean, comment: Option[String],
+                         originalBases: Option[String] = None, originalQuals: Option[String] = None,
+                         readInfo: Option[ReadInfo] = None)
```

Comment:
Ideally case casses should either a) fit all on one line, or b) have a member per line.  E.g. 
```suggestion
  case class DemuxRecord(name: String,
                         bases: String,
                         quals: String,
                         molecularBarcode: Seq[String],
                         sampleBarcode: Seq[String],
                         readNumber: Int,
                         pairedEnd: Boolean,
                         comment: Option[String],
                         originalBases: Option[String] = None,
                         originalQuals: Option[String] = None,
                         readInfo: Option[ReadInfo] = None)
```